packages/reports/tests/unit/consumers/request/sort-test.js

```diff
@@ -50,52 +52,112 @@ test('UPSERT_SORT', function(assert) {
 test('REMOVE_SORT', function(assert) {
   assert.expect(1);
 
-  const removeSortByMetricName = (metric) => {
-    assert.equal(metric,
+  const removeSortByMetricName = (metricName) => {
+    assert.equal(metricName,
       'click',
-      'metricSort is removed from the request');
+      'removeSortByMetricName is called with correct metric name');
   };
 
   let currentModel = { request: {
         removeSortByMetricName
       }},
       consumer = this.subject();
 
-  consumer.send(RequestActions.REMOVE_SORT, { currentModel }, 'click', 'desc');
+  consumer.send(RequestActions.REMOVE_SORT, { currentModel }, 'click');
+});
+
+test('REMOVE_SORT_WITH_PARAM', function(assert) {
+  assert.expect(2);
+
+  let parameters = { currency: 'USD' },
+      revenueUSD = {
+        metric: { name: 'revenue' },
+        parameters,
+        hasParameters: true,
```

Comment:
What shape is this object? [Request Metric](https://github.com/yahoo/navi/blob/fc6bb19c3b376c9309f49911aa1f79c6291e3a65/packages/reports/addon/models/bard-request/fragments/metric.js) or [Metric Metadata](https://github.com/yahoo/navi/blob/fc6bb19c3b376c9309f49911aa1f79c6291e3a65/packages/data/addon/models/metadata/metric.js)?

Reply:
didn't notice we have two models (@balajimanoharan also mentioned it in another comment here), I'll check again and adjust the test and method description accordingly

Reply:
it's Metric Metadata, I'll remove the `canonicalName` property